protocol/messenger_communities.go

```diff
@@ -276,6 +276,27 @@ func (m *Messenger) updateCommunitiesActiveMembersPeriodically() {
 	}()
 }
 
+func (m *Messenger) CheckCommunitiesToUnmute(response *MessengerResponse) {
```

Comment:
It would be good to get at least a basic test coverage for this public function.